### PR TITLE
Balance All Carrier and Other Mobile AntiNuke.

### PR DESCRIPTION
**Pull request by skymyj**
Actually, all mobile Antinuke cost more and have less HP. 
Carrier have better HP and have repair pod.
Turtle : 2250M and 50KE and 3 K HP
Carrier : 1700M and 80KE and 7K HP
Spider : 1500M and 105K and 780 HP

### CODE REVIEW

**Comment by nixtux** [comment]
Please follow https://github.com/techannihilation/TA/issues/294

**Comment by Antarac** [comment]
It's only three examples for so many changes.
And is it only TLL Units as example?
Would be also interesting to compare all three Factions.

On Land site there is Anti Nuke Building and Anti Nuke Units.
Sea got no Anti Nuke Building, only a Anti Nuke Unit.
So only one kind of Anti Nuke and the price is between
building and mobile price which seems not that bad.

Sea base player in team and ffa games often have lesser metal than land players.
Maybe fair in different situations if the Anti Nuke is cheaper for sea base.

Add also a static Anti Nuke for sea bases if that helps?

**Comment by skymyj** [comment]
I think, carrier  should have same range from adv mobile antinuke.

**Comment by skymyj** [comment]
No need to add a new  static unit.

**Comment by Antarac** [comment]
Well, settle range between static and mobile also like the metal cost?

**Comment by skymyj** [comment]
I think, we have increase again metal cost if we increase the range.

**Comment by Antarac** [comment]
If you do not explain
why no static defense,
why increase cost,
why lower range,
it's hard how to work this out.

And by the way, i never used
that repair pod since OTA.
To me they are all useless.
It's only a swimming anti nuke for me.

**Comment by skymyj** [comment]
Repair pod is for gunship air. Very usefull, to make economy. Best use is with fortress Core.
why no static defense => there is already carrier. Nix has too much work to make useless unit.
why increase cost => Cause Adv Mobile antinuke cost 2500M. So if we transform carrier antinuke into Adv antinuke, we should increse metalcost. 
Why adv antinuke => cause sea didn't have adv antinuke. Like kbot for TLL. They didn't have, so turtle are adv antinuke. => we should add ddescription to inform.
why lower range => never speak about lower range.

**Comment by skymyj** [comment]
In seeing in detail all values, the balance is totally unfair.

**Comment by Antarac** [comment]
Repair Pod:
True, i forgot we got only OP players in TA,
not only noobs like in BA... XD
Sounds quiet arrogant ;-P
I do not use it, cause in the past planes always
got stuck on the plattforms, and i need no
figthers that park in the base.
Remembers me to test that.

Static defense:
Static Anti Nuke was just a suggestion.
If it's 2 kinds of anti nuke like on land site,
i could follow you more easy to compare it.
But since it's totally different im not sure
you actually can compare it with land stuff.
People prefer to play on land site mostly,
so if you make water base harder to play,
it get maybe even more worst.

Advanced anti nuke:
You mention that not in beginning.
You bring new theme like advanced anti,
in middle of text, that change whole Issue.

Lower Range:
You want it to have mobile anti nuke range.
It got smallest range of all, doesn't it?
Sounded like you wanna make range smaller.

Now it get a even more complicated issue,
with adding so many changes :-s
You might know the code well, but for me
it means to sit ages here to read and think
and still dont understand the meaning of
the hundreds of variables.
If i would have time i would learn them,
but im afraid i don't have the time.

I give you best feedback i can, cause we
all need feedback for our Issues.
But you manage to make it so big and
complicated that i lose overview =)

**Comment by skymyj** [comment]
Can I have the name of lua file fixing Range ?

**Comment by nixtux** [comment]
Anti range?

**Comment by skymyj** [comment]
I didn't found the file with the anti nuke range ?

**Comment by skymyj** [comment]
for each unit

**Comment by nixtux** [comment]
It's in the weapondef check coverage tag

**Comment by skymyj** [comment]
Thanks. 
Question : did you have ever test turtle underwater to counter nuke ?

**Comment by nixtux** [comment]
Nope need testing, there is a tag for firing underwater, but this really needs testing

**Comment by skymyj** [comment]
So, to resume : 
All Long range antinuke have : 
8000 E and 250 M for 1 missile
Produce -100 (is it logic ?? We should put 0 and increase cost for missile ? No ? )
2320 Range
Time : 60 to add missile

VS Static : No change except HP and more description for armscrab (all terrain...)

**Comment by skymyj** [comment]
For info, carrier was the best OP unit. 
They produced 250 E, cost 1600M, and 75KE, had 7.5 K HP, 2 air repair pod...

File: units/armcarry.lua [hunk 1]
```diff
@@ -21,8 +21,8 @@ return {
 		collisionvolumetype = "CylZ",
 		corpse = "dead",
 		defaultmissiontype = "Standby",
-		description = "Aircraft Carrier with Anti-Nuke",
-		energymake = 250,
+		description = "Aircraft Carrier with long range Anti-Nuke",
+		energymake = -100,
```
Comment (nixtux) [inline units/armcarry.lua]: WTF why this

File: units/corcarry.lua [hunk 2]
```diff
@@ -21,8 +21,8 @@ return {
 		collisionvolumetype = "CylZ",
 		corpse = "dead",
 		defaultmissiontype = "Standby",
-		description = "Aircraft Carrier with Anti-Nuke",
-		energymake = 250,
+		description = "Aircraft Carrier with long range Anti-Nuke",
+		energymake = -100,
```
Comment (nixtux) [inline units/corcarry.lua]: Please No

File: units/tllturtle.lua [hunk 3]
```diff
@@ -4,20 +4,20 @@ return {
...
 		canmove = true,
 		canpatrol = true,
 		canstop = 1,
 		category = "ALL LARGE MOBILE NOTDEFENSE NOTHOVERNOTVTOL NOTSUB NOTSUBNOTSHIP NOTVTOL WEAPON SURFACE",
 		corpse = "dead",
 		defaultmissiontype = "Standby",
-		description = "Mobile Anti-Nuke",
-		energymake = -75,
+		description = "All-terrain and amphibious Long Range Anti-Nuke",
+		energymake = -100,
```
Comment (nixtux) [inline units/tllturtle.lua]: no no no please use energyuse no neg numbers please

**Comment by skymyj** [comment]
I'm for use positive number. 
I just copy the existing units.

**Comment by skymyj** [comment]
All anti-nuke have negative numbers :X

**Comment by skymyj** [comment]
Question : To load a missile, there is a cost. 
So if i understand, anti-nuke cost -100E/S + missile cost (E & M) ?

**Comment by skymyj** [comment]
I sais that cause static antinuke don't use energy if not loading missile.
So, I think, we should make only producing energy.

**Comment by skymyj** [comment]
Changes done. 
So Antinuke produces +25 and adv +50

**Comment by skymyj** [comment]
#417 
Add new values for energy cost.

**Comment by nixtux** [comment]
Sky this can't be mergex

**Comment by nixtux** [comment]
Merged

**Comment by skymyj** [comment]
I try to resolve conflicts

**Comment by skymyj** [comment]
You can now. It was turtle cost.

**Pull request merged**